ananse/peakpredictor.py

```diff
@@ -678,36 +678,43 @@ def predict_factor_activity(self, nregions=20_000):
         try:
             nregions = int(nregions)
         except ValueError:
-            logger.warning("nregions is not an integer, using default number of 20_000")
-            nregions = 20_000
+            nregions = 50_000
+            logger.warning(
+                f"nregions is not an integer, using default number of {nregions}"
+            )
 
         activity = pd.DataFrame()
+        state = np.random.RandomState(567)  # Consistently select same regions
```

Comment:
One random state for all runs? You can move this line into the for loop like `state = np.random.RandomState(567+i)`